src/features/logging/aggregate/index.js

```diff
@@ -123,4 +147,14 @@ export class Aggregate extends AggregateBase {
   queryStringsBuilder () {
     return { browser_monitoring_key: this.agentRef.info.licenseKey }
   }
+
+  /** Abort the feature, once aborted it will not resume */
+  abort (reason = {}) {
+    handle(SUPPORTABILITY_METRIC_CHANNEL, [`Logging/Abort/${reason.sm}`], undefined, FEATURE_NAMES.logging, this.ee)
+    this.blocked = true
+    this.loggingMode = LOGGING_MODE.OFF
+    this.syncWithSessionManager({ loggingMode: this.loggingMode })
```

Comment:
This call will also need to check if session manager exists, so it may make more sense for this method to do the check itself

Reply:
nvm, i see that you already did that 👍